yt_dlp/extractor/nebula.py

```diff
@@ -236,8 +237,8 @@ def _real_extract(self, url):
 
 
 class NebulaClassIE(NebulaBaseIE):
-    IE_NAME = 'nebula:class'
-    _VALID_URL = rf'{_BASE_URL_RE}/(?P<id>[-\w]+)/(?P<ep>\d+)'
+    IE_NAME = 'nebula:class/podcast'
+    _VALID_URL = rf'{_BASE_URL_RE}/(?!myshows|library|videos/)(?P<id>[-\w]+)/(?P<ep>[-\w]+)/?(?:$|[?#])'
```

Comment:
- is the `/` after `videos` intentional? 
- I know this is how the old code does it, and it doesn't actually make a difference here, but it's bugging me: should put a literal `-` at the end of a character class
```suggestion
    _VALID_URL = rf'{_BASE_URL_RE}/(?!myshows|library|videos)(?P<id>[\w-]+)/(?P<ep>[\w-]+)/?(?:$|[?#])'
```